about.md

```diff
@@ -0,0 +1,11 @@
+# How to use the Technology Radar
+
+### Introduction
+
+Edit this file to your needs to provide an introduction to the technology radar. Explain the purpose
```

Comment:
This seems like dummy default text? What is the purpose of this `about.md` file since we already have a `readme.md`?

Reply:
this got auto generated with the upgrade

Reply:
ok. but does our radar still work with this upgrade? we don't really test that in our pipeline at all. may be good to check if our radar is still ok with this upgrade or maybe you did that already?

Reply:
I changed the pipeline to make it work

Reply:
nice!